pinot-spi/src/main/java/org/apache/pinot/spi/config/table/assignment/InstanceAssignmentConfig.java

```diff
@@ -28,29 +28,37 @@
 
 public class InstanceAssignmentConfig extends BaseJsonConfig {
 
+  @JsonPropertyDescription("Configuration for the instance assignment strategy")
+  public final Constants.PartitionSelector _partitionSelector;
```

Comment:
Did we consider making this part of `InstanceReplicaGroupPartitionConfig` itself rather than root `InstanceAssignmentConfig` ?

Reply:
Should probably keep in InstanceAssignmentConfig as the strategy may also be used for non-rg based tables.